tests/test.sh

```diff
@@ -21,6 +21,8 @@ test() {
   RESULT=$?
   if [ $RESULT -ne $expect ]; then
       fail "Failing Test '${name}' (bin)"
+  else
+      echo "Success testing '${name}' (bin)"
```

Comment:
Is it too nit picky to have it say 'Successful Test', to better match the failure comment?